File: python/32b_1353B_TwoArraysAndSwaps.py
def main():
    ans = []
    t = int(input())
    for _ in range(t):
        n, k = input().split()
        n, k = int(n), int(k)
        a = input().split()
        b = input().split()
        for i in range(n):
            a[i] = int(a[i])
            b[i] = int(b[i])
        a.sort()
        b.sort()
        temp = 0
        Sum = 0
        for i in range(n):
            if temp < k and a[i] < b[n - 1 - temp]:
                a[i] = b[n - 1 - temp]
                temp += 1
            Sum += a[i]
        ans.append(Sum)
    for i in ans:
        print(i)

if __name__ == "__main__":
    main()

File: cpp/32b_1353B_TwoArraysAndSwaps.cpp
#include<bits/stdc++.h>

int main(){
    int t, n , k, temp, sum;
    std::vector<int> ans;
    std::cin>>t;
    for (int i = 0; i < t; i++){
        std::cin>>n>>k;
        std::vector<int> a, b;
        sum = 0;
        for(int j = 0; j < n; j++){
            std::cin>>temp;
            a.push_back(temp);
        }
        for(int j = 0; j < n; j++){
            std::cin>>temp;
            b.push_back(temp);
        }
        sort(a.begin(), a.end());
        sort(b.begin(), b.end());
        temp = 0;
        for(int j = 0; j < n; j++){
            if(temp < k && a[j] < b[n - 1 - temp]){
                a[j] = b[n - 1 - temp];
                temp ++;
            }
            sum += a[j];
        }
        ans.push_back(sum);
    }
    for(auto i : ans){
        std::cout<<i<<"\n";
    }
    return 0;
}